tests/components/metoffice/conftest.py

```diff
@@ -0,0 +1,26 @@
+"""Fixtures for Met Office weather integration tests."""
+from unittest.mock import patch
+
+from datapoint.exceptions import APIException
+import pytest
+
+from tests.common import mock_coro
```

Comment:
`mock_coro` is legacy. Please import patch from tests.async_mock. That patch will handle patching both normal functions and coroutines. To mock coroutines explicitly we can use `AsyncMock` if needed.

Reply:
Can't seem to find that import - perhaps my dev branch is out of date. I'll refresh and get back to you...

Reply:
Bother, got the horrible feeling I've messed up my branch (again). Still not particularly expert at this whole Git thing. I shouldn't have merged the latest dev branch onto this branch, should I? It's now re-testing everything as if it was my submission...

Reply:
Okay, Git crisis over (I think) for now. Back onto this tomorrow.